### DISCUSSION TITLE
Malicious Repo Steals Wallet Keys

### DISCUSSION THREAD

**Discussion by rishabhworking** [community/community#113231] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Last month a Buyer with username "parranoe" contacted me on fiverr and shared this Github Repository:

https://github.com/cyrrus717/metabl

He asked me to run the project so he could assess If I am competent to do the job. I ran the project and shared the details with him. While assessing the project I also clicked on the Wallet Connect feature to see if it works. Nothing happened at that moment. Later on the Buyer started to offer me more money saying If I make as many developers as I can to install the source code and run the project. I never did that as I had understood the intentions. He never really answered my questions about the same. Then our communication stopped, never got an order from the guy.

Then without my knowledge, two transaction were made from my address without my knowledge on March 4th at 2:52 AM local time:

https://bscscan.com/tx/0xb1fde735e929ca276361e731baf4bfdfc3ee497d8d289a4c1343b77e12965092
https://bscscan.com/tx/0x635ddff8444fc93fb266748c8f0b651beee5fafaa359d0288e766ee869aab94b

This is the address he used to receive the funds: 0xC8Ae6c5611Ff6B54fA7AFA4d12c0f185cE8881b6
I am sure multiple others have been scammed in the same way. It is my humble request to provide me with any details on this person with github username "cyrrus717", so I can lodge complaints about the same.
![mobileScreenshot](https://github.com/community/community/assets/39197153/ac3def06-05ff-4306-9ec5-6f2bbb5b4db2)
![BscScan2](https://github.com/community/community/assets/39197153/d6562d21-8c34-4fe7-a89d-1cb01121de03)
![BscScan1](https://github.com/community/community/assets/39197153/32b8a0b8-cfea-49b5-a5ec-8abdb60d237c)

**Comment by rishabhworking** [comment]
![fiverr](https://github.com/community/community/assets/39197153/15021400-8819-4e6d-abb0-d3e445efa335)
Fiverr Complaint Ticket

**Comment by AidenJamesYt** [comment]
Several things you can do here: 

1. contact Fiverr Support by opening a ticket.
2. report his repository
3. report his account

**Comment by queenofcorgis** [comment, marked as answer]
Hi @rishabhworking 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
